plugins/org.jkiss.dbeaver.data.office/src/org/jkiss/dbeaver/data/office/export/DataExporterXLSX.java

```diff
@@ -216,15 +214,13 @@ public void init(IStreamDataExporterSite site) throws DBException {
         styleDate.setBorderLeft(border);
         styleDate.setBorderRight(border);
 
-        if (CommonUtils.isEmpty(dateFormat)) {
-            styleDate.setDataFormat((short) 14);
-        } else {
-            styleDate.setDataFormat(wb.getCreationHelper().createDataFormat().getFormat(dateFormat));
-        }
-
         this.rowCount = 0;
         this.sheetIndex = 0;
 
+        String df = CommonUtils.toString(properties.get(PROP_DATE_FORMAT), DEFAULT_DATE_FORMAT);
+        this.dateFormatString = df;
```

Comment:
Why not just assign a value to the field directly?

Reply:
because I have to use it below one more time